pkg/monitor/sqsevent/sqs-monitor.go

```diff
@@ -194,6 +194,7 @@ func (m SQSMonitor) isInstanceManaged(instanceID string) (bool, error) {
 		return false, err
 	}
 	if len(asgs.AutoScalingInstances) == 0 {
+		log.Debug().Str("instance_id", instanceID).Msg("Did not find an Auto Scaling Instance for the given instance id")
```

Comment:
can you change "Auto Scaling instance" to "Auto Scaling Group"

Reply:
Done!